### PR TITLE
Litterpic 290 adding cursor

### PR DESCRIPTION
**Pull request by achitashakhanova24**
There are 2 issues that have been resolved:

The comment input is going to stay open, so the user is able to see the comment they added.  It was done by commenting out the line //setOpenCommentInput(null); so that line is not executed.

Adding left padding to the .comment-text-input resolved the issue of the invisible cursor in the text input.

### CODE REVIEW

File: pages/stories.js [hunk 1]
```diff
@@ -261,10 +261,9 @@ function Stories() {
                 ];
                 setPostComments(updatedPostComments);
 
-                setComments({...comments, [postId]: ''});
-                setOpenCommentInput(null);
+                setComments({...comments, [postId]: '' });
             } catch (error) {
-                console.error('Error adding comment:', error);
+                  console.error('Error adding comment:', error);
```
Comment (LitterPic) [inline pages/stories.js:266]: Please remove the leading spaces which were added in front of console.error, thanks!

Comment (achitashakhanova24) [reply pages/stories.js:266]: Copy that!

Comment (LitterPic) [reply pages/stories.js]: Looks good.

**Review by LitterPic — APPROVED** [review]
Looks good, nice work!

**Pull request merged**